src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { StarRate, Wishlist, Basket } from "@shared";
+import { Link } from "react-router-dom";
+
+interface ProductProp {
+  img: string;
+  name: string;
+  type: string;
+  cost: number;
+  rate: number;
+}
+
+const ProductCardExample: ProductProp = {
+  img: "https://www.collinsdictionary.com/images/full/paper_111691001.jpg",
+  name: "Paper name",
+  type: "Paper type",
+  cost: 1500,
+  rate: 1,
+};
+
+export default function ProductCard ({ img, name, type, cost, rate }: ProductProp & typeof ProductCardExample): JSX.Element {
+  return (
+    <div className="m-1 h-[438px] w-80 rounded-lg border-2 border-black">
```

Comment:
I think we can create default sizes for our objects in tailwing config for different screen sizes. And reuse them